Dockerfile

```diff
@@ -27,8 +27,10 @@ FROM node:12-stretch-slim
 
 WORKDIR /srv/www
 EXPOSE 3000
-ENTRYPOINT npm start
 
-COPY package.json package-lock.json next.config.js ./
+# Note: using npm start here will cause error when exiting the container
+ENTRYPOINT node_modules/.bin/pm2-runtime ecosystem.config.js
```

Comment:
how about use cmd?
https://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/

Reply:
Thanks for advice, changed to `CMD`